root/etc/e-smith/templates/httpd/vhost-extra/30directory20options

```diff
@@ -1,3 +1,2 @@
       Options None
-      Options +Indexes
```

Comment:
We can't change the behavior on current installations because someone may already relay on it.
This fragment must remain unchanged since it already allows directory listing on subfolders.